polkadot/parachain/test-parachains/undying/collator/src/cli.rs

```diff
@@ -81,6 +81,11 @@ pub struct RunCmd {
 	/// we compute per block.
 	#[arg(long, default_value_t = 1)]
 	pub pvf_complexity: u32,
+
+	/// If true, the collator will behave maliciously by submitting
+	/// the same collations to all assigned backing groups.
+	#[arg(long, default_value_t = false)]
+	pub malus: bool,
```

Comment:
we could have this malus flag map to an enum of variants, describing what malicious behaviour it's triggering